app/dashboard/helpers.py

```diff
@@ -433,6 +445,8 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
                 'featuring_date': timezone.make_aware(
                     timezone.datetime.fromtimestamp(metadata.get('featuring_date', 0)),
                     timezone=UTC),
+                'repo_type': metadata.get('repo_type', None),
+                'unsigned_nda': unsigned_nda if unsigned_nda else None,
```

Comment:
Do we need to check for `unsigned_nda` here if its already set to `None`? Could it be something other than a `BountyDocuments` object or `None`?